server/routes/events.js

```diff
@@ -3,7 +3,7 @@ const router = express.Router()
 const db = require('../models/index')
 
 router.get('/', (req, res) => {
-    db.Event.find().populate("organization", "orgname -_id")
+    db.Event.find()//.populate("organization", "orgname -_id")
```

Comment:
Could we put this populate call back?? I was using it to display the organization name that was hosting the event on the search results.